src/libcore/mem.rs

```diff
@@ -1071,7 +1071,7 @@ impl<T: ?Sized> DerefMut for ManuallyDrop<T> {
 /// optimizations, potentially resulting in a larger size:
 ///
 /// ```rust
-/// # use std::mem::{MaybeUninit, size_of, align_of};
+/// # use std::mem::{MaybeUninit, size_of};
```

Comment:
I had to fix this to make the tests pass... makes me wonder why are passing on CI.

Reply:
This was my mistake: https://github.com/rust-lang/rust/pull/59940#issuecomment-495955528.